### DISCUSSION TITLE
Live Stream Delay?

### DISCUSSION THREAD

**Discussion by AlbinCederblad** [gopro/OpenGoPro#242] [Q&A]
I'm using the Hero 9 and have been trying to reduce the live stream delay without success. When streaming via RTMP to an Ant Media Server the lowest i can get the delay to is 1.5-2s, which isn't feasible if I want to be able to have direct feedback communication. In comparison, streaming the front camera from my Android phone via WebRTC has 1/10th of that delay.

There don't seem to be any settings which affect the live streaming delay, in fact, setting lower bitrate and resolution seem to even increase delay. And removing smoothing in settings doesn't seem to have any affect in the live stream mode. And it seems to me that most of this delay is because of some post processing the camera does?

I'm looking for any possible way to reduce delay, and if I can configure that via BLE commands.

Any help/info/suggestions are appreciated :)
Thanks

**Comment by tcamise-gpsw** [comment]
Hi. Yes, this is a common problem that people are currently reporting. This is being tracked in #229.

In short, we need to document this and provide a demo.
